src/pt/pobreflix/src/eu/kanade/tachiyomi/animeextension/pt/pobreflix/extractors/MyStreamExtractor.kt

```diff
@@ -0,0 +1,50 @@
+package eu.kanade.tachiyomi.animeextension.pt.pobreflix.extractors
+
+import eu.kanade.tachiyomi.animesource.model.Video
+import eu.kanade.tachiyomi.lib.playlistutils.PlaylistUtils
+import eu.kanade.tachiyomi.network.GET
+import okhttp3.Headers
+import okhttp3.OkHttpClient
+
+// From animeworldindia
+class MyStreamExtractor(private val client: OkHttpClient, private val headers: Headers) {
+
+    private val playlistUtils by lazy { PlaylistUtils(client, headers) }
+
+    fun videosFromUrl(url: String, language: String): List<Video> {
+        val host = url.substringBefore("/watch?")
+
+        return runCatching {
```

Comment:
the `videoListParse` function is already catching any errors coming from extractors, there's no need to add one here as well